amp.php

```diff
@@ -449,6 +446,7 @@ function amp_force_query_var_value( $query_vars ) {
  * @return void
  */
 function amp_maybe_add_actions() {
+	_deprecated_function( __FUNCTION__, '1.5' );
```

Comment:
The deprecated functions here should be moved to `includes/deprecated.php` (which interestingly is not being included anywhere!)

Also, it would be good to move all of the non-bootstrapping functions in `amp.php` into `includes/amp-helper-functions.php`. This will allow us to use PHP>5.2 in these functions.

Reply:
> which interestingly is not being included anywhere!

Hehe, does that mean that the current contents of that file can be hard-deprecated ?

Reply:
Yes, probably. The file was introduced in 9750909638bc0a9864d76e1a57ef9bd91e35f794 by yours truly but I forgot to include it. So much backwards-incompatibility breakage that didn't happen to anyone!

Reply:
No relevant usages: https://wpdirectory.net/search/01DT14RRMMNGT84GHZGZC900F1